cardano-cli/src/Cardano/CLI/Shelley/Commands.hs

```diff
@@ -162,21 +162,22 @@ renderKeyCmd cmd =
 data TransactionCmd
   = TxBuildRaw
       AnyCardanoEra
-      [TxIn]
+      [(TxIn, Maybe ScriptFile)]
+      -- ^ Transaction inputs with optional spending scripts
       [TxOutAnyEra]
-      (Maybe Value)
-      -- ^ Multi-Asset value
+      (Maybe (Value, [ScriptFile]))
+      -- ^ Multi-Asset value with script witness
       (Maybe SlotNo)
       -- ^ Transaction lower bound
       (Maybe SlotNo)
       -- ^ Transaction upper bound
       (Maybe Lovelace)
       -- ^ Tx fee
-      [CertificateFile]
-      [(StakeAddress, Lovelace)]
+      [(CertificateFile, Maybe ScriptFile)]
+      -- ^ Certificates with potential script witness
+      [(StakeAddress, Lovelace, Maybe ScriptFile)]
       TxMetadataJsonSchema
       [ScriptFile]
-      -- ^ Auxillary scripts
```

Comment:
This comment got lost in the previous edits and rebasing. The comment is a useful reminder of what this is for.